src/components/ArticleComponents/Codebox/index.tsx

```diff
@@ -20,49 +20,53 @@ const replaceLanguages = (language: string) =>
     .replace(/mjs|cjs|javascript/i, 'js')
     .replace(/console|shell/i, 'bash');
 
-const replaceLabelLanguages = (language: string) =>
```

Comment:
Why was this removed? 👀

Reply:
Now it's useless. Because we use other system to display languageLabe.
> I'm not crazy ?

Reply:
Please revert this change. It is still relevant. And should be used inside `{lang.toLowerCase()}` as `{replaceLabelLanguages(lang.toLowerCase())}

Reply:
Yep, i wil do that 😃